### PR TITLE
N21 1823 add other group users feature flag

### PR DESCRIPTION
**Pull request by mrikallab**
# Description
This PR will introduce a new feature flag for activation and deactivation of other group users provisioning.

## Links to Tickets or other pull requests
- https://github.com/hpi-schul-cloud/schulcloud-server/pull/4930
- https://ticketsystem.dbildungscloud.de/browse/N21-1823

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

**Review by IgorCapCoder — APPROVED** [review]

File: ansible/group_vars/all/config.yml [hunk 1]
```diff
@@ -1015,3 +1015,8 @@ configuration_all:
     server: true
     client: false
     nuxtclient: false
+  FEATURE_OTHER_GROUPUSERS_PROVISIONING_ENABLED:
```
Comment (MarvinOehlerkingCap) [inline ansible/group_vars/all/config.yml:1018]: `FEATURE_SCHULCONNEX_OTHER_GROUPUSERS_PROVISIONING_ENABLED`, to be more in line with `FEATURE_SCHULCONNEX_COURSE_SYNC_ENABLED`

File: ansible/group_vars/all/config.yml [hunk 2]
```diff
@@ -1015,3 +1015,8 @@ configuration_all:
...
+    value: false
```
Comment (MarvinOehlerkingCap) [inline ansible/group_vars/all/config.yml:1019]: has to be `value: "false"`

Comment (MarvinOehlerkingCap) [reply ansible/group_vars/all/config.yml:1019]: and should be true in every environment

**Review by MarvinOehlerkingCap — CHANGES_REQUESTED** [review]

File: ansible/group_vars/all/config.yml (hunk 2 above)
Comment (mrikallab) [reply ansible/group_vars/all/config.yml:1019]: Why should be true in every environment ? There is no use of it currently in nuxt or old client..

Comment (MarvinOehlerkingCap) [reply ansible/group_vars/all/config.yml:1019]: environment meaning dev, ref, prod

**Review by MarvinOehlerkingCap — APPROVED** [review]

**Pull request merged**